src/chronos/chronos2/config.py

```diff
@@ -63,6 +65,7 @@ def __init__(
         vocab_size: int = 2,
         pad_token_id: int = 0,
         rope_theta: float = 10000.0,
+        attn_implementation: str | None = None,
```

Comment:
How about we explicitly enumerate all available options?
```suggestion
        attn_implementation: Literal["eager", "sdpa"] = "sdpa",
```